src/commands/branches.ts

```diff
@@ -86,6 +87,12 @@ export const builder = (argv: yargs.Argv) =>
             implies: 'compute',
             default: 0,
           },
+          cu: {
+            describe:
+              'The number of Compute Units. Could be a fixed size (e.g. "2") or a range delimited by a dash (e.g. "0.5-3").',
```

Comment:
This requires proper wording

Reply:
For wording, consider:
"Set your compute size. For a fixed size, enter a single number (e.g., "2"). For autoscaling, enter a range with a dash (e.g., "0.5-3")."